src/main/java/com/google/sps/data/Organization.java

```diff
@@ -18,28 +18,39 @@
 
 public final class Organization {
 
-  private int id;
+  private long id;
   private String name;
   private String email;
   private String address;
   private String description;
   private String phoneNum;
-  private int openHour;
-  private int closingHour;
+  private long openingHour;
```

Comment:
robs comment: Is this openingHour or openingTimestamp? (Same for closingHour)

Reply:
openingHour which is for the organizations opening times. Datastore stores these as longs. Should I change it to int and convert it where needed?

Reply:
Breaking down the question a bit:
 1) What's currently being stored?
 2) Does the current name reflect what's being stored?
 3) What should be stored?

Reply:
1. Currently the organization can input the hour that there organization opens and closes which is being stored in the respective variables
2. I do believe so since for now we are only allowing them to input a general opening hour and a general closing hour. What did you have in mind?
3. In my mind we are storing organization information that will be displayed to the user (a single number and not a timestamp). Are you talking about int vs long or a timestamp?

Reply:
"""In my mind we are storing organization information that will be displayed to the user (a single number and not a timestamp). Are you talking about int vs long or a timestamp?"""

This is exactly what I'm trying to get at. To me as a reader this looked like a unix timestamp (hence a long). If it's an hour of the day: long doesn't make much sense to me.

Int however is also underspecified. Is 11pm ==23? How do you represent the time zone?

Reply:
will go through and deprecate this in another PR that will address these variables in other files